spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -177,9 +177,7 @@ public Table view(
 		modelBuilder.addRow().addValue("Id ").addValue(taskExecutionResource.getExecutionId());
 		modelBuilder.addRow().addValue("Name ").addValue(taskExecutionResource.getTaskName());
 		modelBuilder.addRow().addValue("Parameters ").addValue(taskExecutionResource.getParameters());
-		modelBuilder.addRow().addValue("External Execution Id ").addValue(taskExecutionResource.getExternalExecutionID());
```

Comment:
In this case I'm not sure it fits in that we are getting detail for one specific task and not a list of them.